src/libraries/System.Data.Odbc/ref/System.Data.Odbc.ODBC32.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace System.Data.Odbc
 {
+    // This type is required for Desktop to Core serialization.
```

Comment:
Binary serialization does not use ref assemblies. I do not think we should be touching ref assemblies to fix binary serialization issues. This should be in the implementation assembly only.

Reply:
The other route @ericstj suggested here was https://github.com/dotnet/runtime/pull/37944#discussion_r449387420

> Rather than duplicating this type, let's split out its definition and share the source with the PNSE and the real build.
Make ODBC32 partial, move TypeForwardedFrom to partial file and include RETCODE.

i can try this